test/functional/README.md

```diff
@@ -0,0 +1,372 @@
+## Functional test suite
+
+This directory contains the functional test suite for the IoTA JSON. This test suite is based on mocha and chai. For
+mocks, we use the nock library. Additionally, it uses some specific functions to ease to implement the test. Helper
+functions are located in the `testUtils.js` file.
+
+There are 2 tests files in this directory:
+
+-   `fuctional-tests.js`: This file contains the test defined in the "classic way". This means, coded in the JS file as
+    any other mocha test. It uses the functions defined in the `testUtils.js` file to simplify the tests.
+-   `functional-tests-auto.js`: This file contains the test defined in the "automatic way". This means, the test cases
+    are defined as JSON in a separate file (`testCases.js`). This file is loaded by the test suite and the test cases
+    are automatically generated. This is the recommended way to define the test cases.
+
```

Comment:
```suggestion
    are automatically generated.

The recommended way is to use `testCases.js` (run by `functional-tests-runner.js`). The `fuctional-tests.js` file is provides as basically as an example in the case the "old way" needs to be used.
```

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1525/commits/e6d978198cf097d72ea7f26dcba22915e2f15554 and https://github.com/telefonicaid/iotagent-node-lib/pull/1525/commits/ac1ddf50bf17290a3a57d608e3bc6e603b645403